unit-tests/src/test/scala/java/lang/ProcessTest.scala

```diff
@@ -58,13 +58,13 @@ class ProcessTest {
     val out =
       try {
         unistd.dup2(fd, unistd.STDOUT_FILENO)
-        fcntl.close(fd)
```

Comment:
I think this should include removing the function from `fcntl` since that doesn't seem to be in the spec. That would need to be in the release notes as well if we remove it to point the user to `unistd`.

Reply:
The other option, maybe better is to create another Issue or PR to fix up the `fcntl` so that it matches the spec.

Reply:
Eric,  thank you for the timely and helpful review.

re: removing fcntl.close()   Great Idea!  Reference PR #1633 opened on Jun 22, 2019.
I think you fixed some of the fcntl.close places in your recent posixlib/clib PR.
This fixes another.    I could rebase PR #1633 and see what remains. I think it
will be very little beyond actually nuking fcntl.close().

Reply:
Sounds great.